internal/injector/aspect/advice/call.go

```diff
@@ -42,6 +42,7 @@ func (a *appendArgs) Apply(ctx context.AdviceContext) (bool, error) {
 		if err != nil {
 			return false, err
 		}
+		ctx.EnsureMinGoLang(t.Lang)
```

Comment:
From what I see all 4 advice implementation where you added `ctx.EnsureMinGoLang(t.Lang)` have the CodeCov warning onto them. Is this expected?

Reply:
That is not expected (and not the case anymore AFAICT).